ddtrace/tracer/writer.go

```diff
@@ -183,15 +184,21 @@ func (h *logTraceWriter) encodeSpan(s *span) {
 	first := true
 	for k, v := range s.Meta {
 		if first {
-			h.buf.WriteByte('"')
 			first = false
 		} else {
-			h.buf.WriteString(`,"`)
+			h.buf.WriteString(`,`)
 		}
-		h.buf.WriteString(k)
-		h.buf.WriteString(`":"`)
-		h.buf.WriteString(v)
-		h.buf.WriteString(`"`)
+		m, err := json.Marshal(k)
+		if err != nil {
+			log.Error("error encoding meta key: %v", err)
+		}
+		h.buf.Write(m)
+		h.buf.WriteString(":")
+		m, err = json.Marshal(v)
+		if err != nil {
+			log.Error("error encoding meta value: %v", err)
```

Comment:
```suggestion
			log.Error("Error encoding meta value %q: %v", v, err)
```
Log messages need to be sentences with capital letter.